python/dgl/graph.py

```diff
@@ -1137,7 +1141,8 @@ def from_networkx(self, nx_graph, node_attrs=None, edge_attrs=None):
         Parameters
```

Comment:
Describe if the reordering happens, what ordering will be used. From nx: "inherit node ordering from G.nodes() “sorted” : inherit node ordering from sorted(G.nodes())"